distributions/openhab/src/main/resources/bin/backup

```diff
@@ -55,9 +55,15 @@ setup(){
 
```

Comment:
Don't forget to add your new feature in the `--help` documentation above (Lines 17-18) ;)

Reply:
Oh. Never noticed there was a built in documentation. Fixing.